test/export/test_export.py

```diff
@@ -6720,7 +6722,6 @@ def forward(self):
         ep = export(m, ())
         self.assertEqual(ep.graph_signature.lifted_tensor_constants, ["x"])
 
-    @testing.expectedFailureRetraceability  # T186979579
```

Comment:
Why does this error disappear now? Does this mean we should also have a variant that is Retraceability w/ Strict export?

Reply:
Ah, because of a bug in my code, I forgot to update the strict flag in retracing export call. Changed.